build.gradle

```diff
@@ -387,6 +387,8 @@ dependencies {
     // make sure the dependencies use the latest version
     implementation "com.google.code.gson:gson:2.10.1"
 
+    implementation "com.flipkart.zjsonpatch:zjsonpatch:${zjsonpatch_version}"
```

Comment:
is this new dependency really needed?

Reply:
I initially added this to be able to properly annotate patch types and to apply patches to database models. That is not needed anymore, so I removed this dependency.